darkreader.css

```diff
@@ -27626,7 +27626,8 @@ textarea.c-input_textarea.c-input_textarea--with_hint {
 
 /* Windows-specific header/footer */
 .p-classic_nav__channel_header,
-.p-classic_nav__right_header.p-classic_nav__right_header--whats_new_showing,
+.p-classic_nav__right_header,
+.p-classic_nav__right_header--whats_new_showing,
```

Comment:
This looks like the original CSS was just missing a comma to allow these two classes to both be edited. Adding it fixed the search bar's white block.